libs/sr/formula/src/data/lightCone/sheets/ASecretVow.ts

```diff
@@ -35,20 +34,25 @@ const sheet = register(
   registerBuff(
     'cond_dmg_',
     ownBuff.premod.dmg_.add(
-      boolConditional.ifOn(
-        cmpGE(lcCount, 1, subscript(superimpose, dm.cond_dmg_))
+      cmpGE(
```

Comment:
Is cmpGE more expensive than the conditional check? I wonder which one in the outside is better?

Reply:
not sure if any is more expensive, but cmpGE outside allows better/easier debug filtering